dirsrvtests/tests/suites/backups/backup_test.py::test_replication is failing

**Issue Description**
A clear and concise description of what the bug is.

**Package Version and Platform:**
 - Platform:  Fedora
 - Package and version: main branch
 - Browser [e.g. chrome, safari]

**Steps to Reproduce**
Steps to reproduce the behavior:
1. Looks at lmdb backup CI test results
 For example: https://github.com/389ds/389-ds-base/actions/runs/17801804079/job/50604701871?pr=6999
2. See error:
 Test fail because restore aborted
ERROR    lib389:tasks.py:720 Error: restore task restore_2025-09-17T15:25:59.844606 exited with 53


**Expected results**
Test should pass

**Screenshots**

**Additional context**
This looks like a regression

--- a/ldap/servers/slapd/back-ldbm/db-mdb/mdb_import_threads.c
+++ b/ldap/servers/slapd/back-ldbm/db-mdb/mdb_import_threads.c
@@ -3737,8 +3737,8 @@ out:
 }
 
 /*
- * Helper function that remove * some attributes (that may change)
- * before comparing config entries
+ * Helper function that removes attributes which may change,
+ * to ensure accurate comparison of config entries.
  */
 static void
 clear_recomputed_attributes(Slapi_Entry *entry)

--- a/ldap/servers/slapd/back-ldbm/db-mdb/mdb_import_threads.c
+++ b/ldap/servers/slapd/back-ldbm/db-mdb/mdb_import_threads.c
@@ -3737,6 +3737,17 @@ out:
 }
 
 /*
+ * Helper function that remove * some attributes (that may change)
+ * before comparing config entries
+ */
+static void
+clear_recomputed_attributes(Slapi_Entry *entry)
+{
+    entry_replace_values(entry, CONFIG_INSTANCE_CACHEMEMSIZE, NULL);
+    entry_replace_values(entry, CONFIG_INSTANCE_DNCACHEMEMSIZE, NULL);
+}
+
+/*
  * read the backed up index configuration
  * adjust them if the current configuration is different from it.
  * this function is called from dblayer_restore (archive2ldbm)
@@ -3770,6 +3781,12 @@ dbmdb_dse_conf_verify_core(struct ldbminfo *li, char *src_dir, char *file_name, 
         goto out;
     }
 
+    for (Slapi_Entry **entry=curr_entries; *entry; entry++) {
+        clear_recomputed_attributes(*entry);
+    }
+    for (Slapi_Entry **entry=backup_entries; *entry; entry++) {
+        clear_recomputed_attributes(*entry);
+    }
     if (0 != slapi_entries_diff(backup_entries, curr_entries, 1 /* test_all */,
                                 log_str, force_update, li->li_identity)) {
         if (force_update) {

